src/ldp/http/metadata/BasicMetadataSerializer.ts

```diff
@@ -0,0 +1,26 @@
+import type { HttpResponse } from '../../../server/HttpResponse';
+import type { RepresentationMetadata } from '../../representation/RepresentationMetadata';
+import { MetadataSerializer } from './MetadataSerializer';
+import type { MetadataWriter } from './MetadataWriter';
+
+/**
+ * Basic serializer that calls all its {@link MetadataWriter}s sequentially.
+ */
+export class BasicMetadataSerializer extends MetadataSerializer {
+  private readonly writers: MetadataWriter[];
+
+  public constructor(writers: MetadataWriter[]) {
+    super();
+    this.writers = writers;
+  }
+
+  public async canHandle(): Promise<void> {
+    // Supports all inputs
+  }
+
+  public async handle(input: { response: HttpResponse; metadata: RepresentationMetadata }): Promise<void> {
```

Comment:
```suggestion
  public async handle({ response: HttpResponse; metadata: RepresentationMetadata }): Promise<void> {
```

(if allowed)

Reply:
It needs to be something like `{ response, metadata } : { response: HttpResponse; metadata: RepresentationMetadata }` which is the reason I usually don't do it since it can get quite long with multiple parameters. Could work here

Reply:
Ah it's alright then.